### ISSUE TITLE
Adding conditions as part of spells, items, weapons.

### ISSUE THREAD

**Issue by mcenal** [ThaumRystra/DiceCloud#172]
Be able to add conditions as part of spells, items, weapons.

This would allow for items, weapons, and spells to add those statuses to character sheets and others. So then I could apply a status to my sheet (or if fully integrated team on Dicecloud I could apply to another character sheet of NPC or other Party Member) and then that status cannot be cleared until I apply an item/spell or someone else does that then clears that status.

**Comment by mcenal** [comment]
Currently in V1 that is not possible. 

Buffs can only add Effects, Attacks and Proficiencies.
There is no system in Dicecloud that allows to add specifically a Condition, according to the variable list you cannot address it either as part of a formula. I would like to be able to associate with that Condition menu.

For example: There are certain spells that turn a target blind. I cannot add a Named Buff that would say the target is blinded.  
Attack Rolls against have advantage and the target's attack rolls have disadvantage. There is no system that would be able to add that to a character but adding the Condition it would handle all those details.

**Comment by ThaumRystra** [comment]
Conditions are just buffs that have a default set of values. You can add a buff with the identical values that the conditions have, and they will be indistinguishable once applied.

So I understand that the feature request here is to add a way of adding those default buffs to things that can apply buffs without needing to duplicate them manually?

**Comment by ThaumRystra** [comment]
things are done a bit differently in #184, once the buff/condition system is complete there, open a new issue if it is still not implemented suitably
